distros/humble/overrides.nix

```diff
@@ -30,6 +39,12 @@ in with lib; {
 
   gazebo = self.gazebo_11;
 
+  gazebo-ros = rosSuper.gazebo-ros.overrideAttrs ({
+    nativeBuildInputs ? [], ...
+  }: {
+    nativeBuildInputs = nativeBuildInputs ++ [ self.qt5.qtbase ];
```

Comment:
Shouldn't this be just `buildInputs`? It makes the difference in cross compilation.

Reply:
I think you are right. We will look at this on Monday.

Reply:
We changed it and it worked. Thanks